### ISSUE TITLE
Why the count of nodes is different when exporting the onnx model from 1.27.0 and 1.28.0

### ISSUE THREAD

**Issue by wminiboy** [quic/aimet#2497]
Why the count of nodes is different when exporting the onnx model from 1.27.0 and 1.28.0, but the model is same.

When use 1.27.0, the log  show:
2023-10-10 12:18:06,948 - Utils - INFO - successfully created onnx model with 1894/1899 node names updated                                                                                                 
2023-10-10 12:18:10,102 - Quant - INFO - Layers excluded from quantization: []   

But when use 1.28.0, the log show:
2023-10-10 12:13:53,586 - Utils - INFO - successfully created onnx model with 854/1517 node names updated
2023-10-10 12:13:56,234 - Quant - INFO - Layers excluded from quantization: []

Is there an impact on the results of qat for 1.28.0 ?

**Comment by quic-mangal** [comment]
1.28 uses PyTorch 1.13 which has significantly changed the ONNX exported model.

**Comment by wminiboy** [comment]
@quic-mangal  Exactly, my question now is, will it have an impact on qat, Especially for encondings files ?  Or it has no impact on qat, only on the export of the onnx model.  Thanks !

**Comment by quic-hitameht** [comment]
@wminiboy As @quic-mangal pointed out that the pytorch 1.13 version applies bunch optimizations to ONNX graph and provides more optimized graphs than pytorch 1.9 version. With that being said, this should not have an impact on QAT. Let us know if you see any issues with AIMET 1.28 release.

**Comment by wminiboy** [comment]
@quic-hitameht Thanks for your reply.   When I export the model on 1.28.0, the log show as follows:

....../anaconda3/envs/aimet-1.28.0/lib/python3.8/site-packages/aimet_torch/onnx_utils.py:239: FutureWarning: 'torch.onnx._patch_torch._graph_op' is deprecated in version 1.13 and will be removed in version 1.14. Please note 'g.op()' is to be removed from torch.Graph. Please open a GitHub issue if you need this functionality..
  return g.op('aimet_torch::CustomMarker', inp, id_s=identifier, leaf_s=is_leaf, start_s=start)

....../anaconda3/envs/aimet-1.28.0/lib/python3.8/site-packages/torch/onnx/_patch_torch.py:81: UserWarning: The shape inference of aimet_torch::CustomMarker type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function. (Triggered internally at ../torch/csrc/jit/passes/onnx/shape_type_inference.cpp:1884.)
  _C._jit_pass_onnx_node_shape_type_inference(

....../anaconda3/envs/aimet-1.28.0/lib/python3.8/site-packages/torch/onnx/utils.py:687: UserWarning: The shape inference of aimet_torch::CustomMarker type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function. (Triggered internally at ../torch/csrc/jit/passes/onnx/shape_type_inference.cpp:1884.)
  _C._jit_pass_onnx_graph_shape_type_inference(

....../anaconda3/envs/aimet-1.28.0/lib/python3.8/site-packages/torch/onnx/utils.py:1178: UserWarning: The shape inference of aimet_torch::CustomMarker type is missing, so it may result in wrong shape inference for the exported graph. Please consider adding it in symbolic function. (Triggered internally at ../torch/csrc/jit/passes/onnx/shape_type_inference.cpp:1884.)

2023-10-11 11:23:10,337 - Utils - INFO - successfully created onnx model with 434/773 node names updated  

As a result, when convert the exported model  by qnn,  the quantified model can run on 8550 , but the results are very poor .   However, the model exported with 1.27.0 can work after quantification, although the results are not very good.

**Comment by quic-bharathr** [comment]
Hi @wminiboy While the source code remains compatible with PyTorch 1.9, the 1.28 release wheel package files support PyTorch 1.13 only. We will need to rebuild the package for PyTorch 1.9. We make these packages available for the 1.28 OS release shortly.

**Comment by wminiboy** [comment]
Hi @quic-bharathr Thank for you reply,  Waiting for your good news !

**Comment by wminiboy** [comment]
@quic-bharathr   Hi, How's the progress going?

**Comment by quic-mangal** [comment]
Hi @wminiboy, this is still being looked at. Thanks

**Comment by quic-muchhsu** [comment]
Hi @wminiboy can you try out aimet 2.0 and reopen this issue if not solved.
